frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -433,6 +433,8 @@ export const keyMapping: KeyMappingInterface = {
     },
 }
 
+export const hiddenFilters: Array<string> = ['$test_environment']
```

Comment:
I think @timgl mentioned you can actually just use the existing logic for this, there's a `hide` property (see line 281) that handles this behavior.

Reply:
Makes sense to me.